### PR TITLE
Add json-server and react-query

### PR DESCRIPTION
**Pull request by Lenkly**
I build a fake api with a database that holds just the information I need and set up a json-server. I used react-query to fetch the data and put it on the details site. 
My proxy is not yet running (🧐😞), so I need to let my server run on localhost:4000 for the moment. (Which works.) The artists aren't displayed right yet, that's what I'm working on right now.

### CODE REVIEW

File: client/src/components/Festivals.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+const LineUp = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  font-size: 20px;
+  margin-bottom: 20px;
+  font-size: 12px;
+  font-family: 'Quicksand Book';
+  text-transform: uppercase;
+`;
+
```
Comment (nadinho) [inline client/src/components/Festivals.js:41]: I'm wondering why you are just working with divs and don't use h1, h2, p, etc.?

Comment (Lenkly) [reply client/src/components/Festivals.js:41]: good question actually. I think there's a need to refactor anyways, so I put that on my todo list

File: index.js [hunk 2]
```diff
@@ -1,13 +1,29 @@
...
+
+// const express = require('express');
+// const path = require('path');
+// const app = express();
+// const port = process.env.PORT || 8080;
+
+// app.use(express.static(path.join(__dirname, 'client/build')));
+// app.get('*', (require, response) => {
+//   response.sendFile(path.join(__dirname, 'client/build/index.html'));
+// });
```
Comment (lmachens) [inline index.js:25]: These routes are important for the production build. Instead of `app`, it's `server` now.

File: index.js [hunk 3]
```diff
@@ -1,13 +1,29 @@
...
-app.listen(port, () => {
-  console.log('Server is running');
+server.use(
+  jsonServer.rewriter({
+    '/api/*': '/$1',
+  })
+);
+server.use(middlewares);
+server.use(router);
+server.listen(4000, () => {
```
Comment (lmachens) [inline index.js:13]: Your proxy (client/package.json) targets port 8080. You changed the port to 4000 which is the reason the proxy doesn't work.

**Pull request merged**